deploy/README_HELM.md

```diff
@@ -0,0 +1,343 @@
+# Deploying TopoLVM
+
+> :memo:
+>
+> This document is an adaptation of the existing README.md to Helm Chart.
+> After removing the kustomize manifests, this document will be renamed to README.md.
+
+Each of these steps are shown in depth in the following sections:
+
+1. Deploy [lvmd][] as a `systemd` service on a worker node with LVM installed.
+1. Prepare [cert-manager][] for [topolvm-controller][]. You may supplement an existing instance.
+1. Determine how [topolvm-scheduler][] to be run:
+    - If you run with a managed control plane (such as GKE, AKS, etc), `topolvm-scheduler` should be deployed as Deployment and Service
+    - `topolvm-scheduler` should otherwise be deployed as DaemonSet in unmanaged (i.e. bare metal) deployments
+1. Apply remaining manifests for TopoLVM from `deploy/manifests/base` plus overlays as appropriate to your installation.
```

Comment:
This is not applicable to helm.

Reply:
refs: https://github.com/topolvm/topolvm/pull/325#discussion_r672746413